spec/index.html

```diff
@@ -1016,35 +1016,70 @@ <h3>IRIs</h3>
         but is based on a processing model important for implementation
         within web browsers and are not described using an [[ABNF]] grammar.
       </p>
+    </div>
 
-      <p><strong>Relative IRI references:</strong>
+    <section id="reference-iris" class="informative">
+      <h4>RDF Reference IRIs</h4>
+      <p>
+        This section is advice to data publishers.
+      </p>
+      <p>
+        <a>IRIs</a> are used to denote <a>resources</a> and should identify the same
+        resource regardless of where the IRI reference is used.
+        [[RFC3987]] defines a syntax that can express
+        IRIs which do not meet the requirement of being a global reference.
+        URI schemes add addition requirements; for example, the 
+        <a data-cite="rfc7230#section-2.7.1">HTTP URI scheme</a> defines 
+        <a data-cite="rfc7230#http-URI">`http-URI`</a>, which
+        requires the presence of a non-empty host name, and, as a consequence, 
+        the path component will start with `/`. 
+        The [[RFC3987]] syntax permits `http:abcd` and `http:///abcd` 
+        which do not meet the HTTP URI scheme.
+      </p>
+      <p>
+        An <dfn data-lt="RDF reference">RDF Reference IRI</dfn>,
+        sometimes just <a>RDF Reference</a>,
+        is an <a>IRI</a> that is suitable for use as a global reference.
+      </p>
+      
+      <p>
+        <b>Reference Resolution:</b>
+        An <a>RDF Reference IRI</a> is unchanged by
+        <a data-cite="RFC3986#section-5">reference resolution</a>. 
+        The <a data-cite="RFC3986#section-3.3">path component</a> 
+        starts with a `/` character and does not contain `.` or `..` segments.
+      </p>
+      <p>
+        <strong>Relative IRI references:</strong>
```

Comment:
It seems like `Reference Resolution:` should have the same semantic importance and styling as `Relative IRI references:`, so either both should be `<b>` or both should be `<strong>`. I think the latter is *probably* better, so that's my suggestion here.
```suggestion
        <strong>Reference Resolution:</strong>
        An <a>RDF Reference IRI</a> is unchanged by
        <a data-cite="RFC3986#section-5">reference resolution</a>. 
        The <a data-cite="RFC3986#section-3.3">path component</a> 
        starts with a `/` character and does not contain `.` or `..` segments.
      </p>
      <p>
        <strong>Relative IRI references:</strong>
```